generated/rflx-enumeration-generic_message.adb

```diff
@@ -43,9 +45,7 @@ is
          when F_Initial =>
             (case Fld is
                   when F_Priority =>
-                     True,
-                  when others =>
-                     False),
+                     True),
```

Comment:
Indent case expressions according to Componolit/ada-style?

Reply:
Fixed by ebf0001.